content/en/universal_service_monitoring/_index.md

```diff
@@ -63,6 +63,36 @@ If you have feedback about what platforms and protocols you'd like to see suppor
 Enable Universal Service Monitoring in your Agent by using one of the following methods depending on how your service is deployed and your Agent configured:
 
 {{< tabs >}}
+{{% tab "Operator" %}}
+
+To enable Universal Service Monitoring with the [Datadog Operator][1], update your `datadog-agent.yaml` manifest. In the `DatadogAgent` resource, set `spec.features.usm.enabled` to `true`:
+
+   ```yaml
+   apiVersion: datadoghq.com/v2alpha1
+   kind: DatadogAgent
+   metadata:
+     name: datadog
+   spec:
+     global:
+       credentials:
+        apiSecret:
+           secretName: datadog-secret
+           keyName: api-key
+        appSecret:
+         secretName: datadog-secret
+         keyName: app-key
+     features:
+       usm:
+         enabled: true
+   ```
+
+**Note:** Datadog Operator v1.0.0 or greater is required.
+
+<div class="alert alert-info">Support for GKE clusters running Google Container-Optimized OS (COS) is coming soon. An upcoming feature of Datadog Operator is introspection to automatically detect the cluster type, allowing for seamless configuration and greater ease of use.</div>
+
+[1]: https://github.com/DataDog/datadog-operator
```

Comment:
not being used

Reply:
The link is used on [line 68](https://github.com/DataDog/documentation/pull/17453/files#diff-d13aba713ff0e6f82271659d00de9d22fe613f3c938e5d2a5125ba09a6bbeb4cR68)

Reply:
sorry, my bad